packages/ketcher-react/src/script/ui/views/modal/components/process/Miew/Miew.tsx

```diff
@@ -117,7 +122,7 @@ const MiewDialog = ({
     (miew: MiewAsType) => {
       miewRef.current = miew
       const factory = new FormatterFactory(server)
-      const service = factory.create('cml')
+      const service = factory.create('cml' as SupportedFormat)
```

Comment:
Could you please also change to `SupportedFormat.cml`?

Reply:
Fixed